### ISSUE TITLE
False positive for tests in bicep: CKV_AZURE_4, CKV_AZURE_5, CKV_AZURE_7, CKV_AZURE_8

### ISSUE THREAD

**Issue by JonasCordsen** [bridgecrewio/checkov#4612]
**Describe the issue**
false positive for tests CKV_AZURE_4, CKV_AZURE_5, CKV_AZURE_7, CKV_AZURE_8

**Examples**
Here is the bicep code
 Code made shorter.
```
resource aks 'Microsoft.ContainerService/managedClusters@2022-11-01' = {
  properties: {
    enableRBAC: true
    }
    addonProfiles: {
      omsAgent: {
        enabled: true
        config: {
          logAnalyticsWorkspaceResourceId: logAnalyticsWorkspaceId
        }
      }
      azurePolicy: {
        enabled: true
        config: {
          version: 'v2'
        }
      }
      kubeDashboard: {
        enabled: false
      }
    }
    networkProfile: {
      networkPlugin: 'azure'
      networkPolicy: 'azure'
    }
  }
}
```

**Version (please complete the following information):**
 - checkov v2.1.244

**Comment by gruebel** [comment]
hey @JonasCordsen thanks for reaching out.

I tested it and only `CKV_AZURE_4` is failing, which is related to the way `omsAgent` is written, we expect it to be all lower case `omsagent`. This looks like a valid case and I can create a PR for it.

I recommend you to update your `checkov` version to the latest one (currently `2.3.59`) and keep it up-to-date.

```shell
Check: CKV_AZURE_4: "Ensure AKS logging to Azure Monitoring is Configured"
        FAILED for resource: Microsoft.ContainerService/managedClusters.aks
        File: //example.bicep:1-26
        Guide: https://docs.bridgecrew.io/docs/bc_azr_kubernetes_1

                1  | resource aks 'Microsoft.ContainerService/managedClusters@2022-11-01' = {
                2  |   properties: {
                3  |     enableRBAC: true
                4  |     addonProfiles: {
                5  |       omsAgent: {
                6  |         enabled: true
                7  |         config: {
                8  |           logAnalyticsWorkspaceResourceId: logAnalyticsWorkspaceId
                9  |         }
                10 |       }
                11 |       azurePolicy: {
                12 |         enabled: true
                13 |         config: {
                14 |           version: 'v2'
                15 |         }
                16 |       }
                17 |       kubeDashboard: {
                18 |         enabled: false
                19 |       }
                20 |     }
                21 |     networkProfile: {
                22 |       networkPlugin: 'azure'
                23 |       networkPolicy: 'azure'
                24 |     }
                25 |   }
                26 | }

```
